CenteredCollectionView/Classes/CenteredCollectionViewFlowLayout.swift

```diff
@@ -148,6 +162,8 @@ private extension CenteredCollectionViewFlowLayout {
 			origin = CGPoint(x: proposedContentOffset.x, y: collectionView.contentOffset.y)
 		case .vertical:
 			origin = CGPoint(x: collectionView.contentOffset.x, y: proposedContentOffset.y)
+        default:
+            origin = .zero
```

Comment:
The spacing here doesn't match the rest of the code. Would you mind making the stacking consistent with the rest of the file?